### ISSUE TITLE
pointerEvents property for item

### ISSUE THREAD

**Issue by khurramraheel** [paperjs/paper.js#780]
It seems we can't use equivalent of '"pointerEvents:none" on paperjs items

**Comment by lehni** [comment]
You mean to prevent an item to receive mouse events even if it has handlers installed?

**Comment by khurramraheel** [comment]
Yeah, if the element is on top of another element and I want to disable its event capture so that I can click on the bottom element.

**Comment by lehni** [comment]
Yes that would be useful indeed. I shall think of a good way of adding this.

**Comment by JulianGarciaCastillo** [comment]
So did you guys find an alternative to use pointerEvents, to prevent mouse events in paper js?

**Comment by lehni** [comment]
I think setting the `item.guide` property to true should have this effect, actually. This is a left-over from Scriptographer times, that we may want to rethink. But for now, that's how you should be able to achieve this.
